superdesk/core/config.py

```diff
@@ -53,12 +53,15 @@ def create_from_dict(cls, config: Dict[str, Any], prefix: Optional[str] = None,
         config_instance._loaded = True
         return config_instance
 
-    def load_from_dict(self, config: Dict[str, Any], prefix: Optional[str] = None, freeze: bool = True):
+    def load_from_dict(
+        self, config: Dict[str, Any], prefix: Optional[str] = None, freeze: bool = True, additional: dict | None = None
```

Comment:
nit:
```suggestion
        self, config: Dict[str, Any], prefix: Optional[str] = None, freeze: bool = True, additional: Optional[Dict] = None
```

Reply:
Based on the [docs](https://docs.python.org/3/library/typing.html#typing.Optional): `Optional[X] is equivalent to X | None (or Union[X, None])`.

IMO we should use type definitions from the language without import from the typing module, where applicable.